src/Icons/src/Iconify.php

```diff
@@ -68,13 +81,25 @@ public function fetchSvg(string $prefix, string $name): string
     {
         $content = $this->http
             ->request('GET', sprintf('/%s/%s.svg', $prefix, $name))
-            ->getContent()
-        ;
+            ->getContent();
 
         if (!str_starts_with($content, '<svg')) {
             throw new IconNotFoundException(sprintf('The icon "%s:%s" does not exist on iconify.design.', $prefix, $name));
         }
 
         return $content;
     }
+
+    private function fetchCollections(): array
+    {
+        $response = $this->http->request('GET', '/collections');
+        if (200 !== $response->getStatusCode()) {
+            throw new \RuntimeException('Could not fetch icon collections from iconify.design.');
+        }
+        if (null === $collections = $response->toArray()) {
```

Comment:
`toArray` never returns null

Reply:
Omg what did wrote ?

(the idea behind this PR was to prevent some unhandled 404 (when the icon set does not exists).. but with your refacto this is probably no more usefull)

Reply:
We can close it now i think ?